examples/apiary_scenarios/asset_request_workflow.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * This file is part of the Ingram Micro Cloud Blue Connect SDK.
+ *
+ * @copyright (c) 2020. Ingram Micro. All Rights Reserved.
+ */
+const { ConnectClient, Fulfillment } = require('@cloudblueconnect/connect-javascript-sdk');
+const fetch = require('node-fetch');
+
+/* eslint-disable no-console */
+
+/**
+ * This module is an example of fulfillment flow between
+ * Cloudblue Connect and Vendor System.
+ * This connect with the Vendor System using an
+ * example of API Vendor System implemented in apiary.io
+ * and documented in the Cloudblue Connect Documentation Portal:
+ * https://connect.cloudblue.com/community/sdk/vendor-scenario-example/fulfillment/
+ *
+ * This module creates a tenant in the Vendor System when is created in Cloudblue Connect.
+ *
+ * @category Operations
+ */
+
+const client = new ConnectClient(
+  'https://api.connect.cloudblue.com/public/v1',
+  'ApiKey SU-069-215-193:e3ae33ea2a7ec62b434452d4feb92eb32dbf57d3',
```

Comment:
shall not be here, also since is used in multiple places, better to move to a configuration file?